### PR TITLE
Minor docs updates

### PR DESCRIPTION
**Pull request by kasperski95**

### CODE REVIEW

**Comment by kasperski95** [comment]
@Arcticae If approved, you can merge it.

File: website/docs/tutorials/03-project-initialization.md [hunk 1]
```diff
@@ -29,10 +29,10 @@ The result of running `protostar init` is a configuration file `protostar.toml`,
 ### Project configuration
 ```toml title="'protostar.toml' is required"
 ["protostar.config"]
-protostar_version = "0.1.0"
+protostar_version = "0.5.0"
```
Comment (mkaput) [inline website/docs/tutorials/03-project-initialization.md:32]: Is it possible to populate this in docs build time, reading from `pyproject.toml`?

Comment (kasperski95) [reply website/docs/tutorials/03-project-initialization.md:32]: Hmm, it should be possible. We could use keep protostar version in the `docusaurus.config.js` and update that file on build time.

Comment (mkaput) [reply website/docs/tutorials/03-project-initialization.md:32]: If you can put it in `docusaurus.config.js` then you cal also read it from `pyproject.toml`. My general motivation is to keep version number in one place.

Comment (kasperski95) [reply website/docs/tutorials/03-project-initialization.md:32]: https://github.com/software-mansion/protostar/issues/951

**Pull request merged**